sirepo/package_data/static/json/schema-common.json

```diff
@@ -150,6 +150,7 @@
         "simulationData": "/simulation/<simulation_type>/<simulation_id>/<pretty>/?<section>",
         "simulationFrame": "/simulation-frame/<frame_id>",
         "simulationSchema": "/simulation-schema",
+        "statelessCompute": "/stateless-compute",
```

Comment:
"t" comes after "r". C-x C-l in our emacs will sort a region.

Reply:
e80129335